index.js

```diff
@@ -133,11 +133,13 @@ var createAssertion = function createAssertion(type, message, wrappers, block, m
 	if (descriptors.length === 0) {
 		throw new RangeError(inspect(type) + ' called with no wrappers defined');
 	}
-	var describeMsgs = [];
-	forEach(wrappers, function (wrapper) {
-		checkThis(wrapper);
-		describeMsgs.push(getThisDescription(wrapper));
```

Comment:
`.use` should take a function that returns a descriptor, not take a descriptor directly.  That will prevent having any subwrappers.